digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -360,6 +361,15 @@ Returns empty result as Json message, not as page not found
       halt(NotAcceptable("Invalid Mass Transit Stop direction"))
   }
 
+
+  private def validatePropertiesMaxSize(properties: Seq[SimpleProperty]) = {
+    val propertiesWithMaxSize: Map[String, Int] = massTransitStopService.getPropertiesWithMaxSize()
+    val invalidPropertiesDueMaxSize: Seq[SimpleProperty] = properties.filter { property =>
+      propertiesWithMaxSize.contains(property.publicId) && property.values.nonEmpty && property.values.forall { value => value.propertyValue.length > 3 }

```

Comment:
I changed this before this comment - it was not checking if the property was empty

Reply:
This is not validating for all the properties instead of for the propertiesWithMaxSize?